src/transformers/trainer.py

```diff
@@ -869,7 +871,6 @@ def num_examples(self, dataloader: DataLoader) -> int:
     def _hp_search_setup(self, trial: Union["optuna.Trial", Dict[str, Any]]):
         """HP search setup code"""
         self._trial = trial
-
```

Comment:
Let's leave the white lines pretty please :-)

Reply:
will do!